src/script/api/script_list.cpp

```diff
@@ -12,9 +12,13 @@
 #include "script_controller.hpp"
 #include "../../debug.h"
 #include "../../script/squirrel.hpp"
+#include <../squirrel/sqvm.h>
```

Comment:
Using `../` with `#include <>` looks wrong to me.

Reply:
That's how all the existing squirrel include in /src/script/ are.

Reply:
Most likely by error. A while ago we fixed a bunch of those `""` vs `<>` for squirrel. Guess those were missed :)

Reply:
It doesn't compile with the `../` removed.
In CMakeLists.txt there is `include_directories(${CMAKE_SOURCE_DIR}/src/3rdparty/squirrel/include)`, but 
`src/script/squirrel.cpp` wants includes in `src/3rdparty/squirrel/squirrel/`.